_pages/security/dynamic-scanning.md

```diff
@@ -5,29 +5,57 @@ parent: Security
 
 [OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project) is the tool we use for doing automated vulnerability scanning.
 
+![zap home screen](../../assets/images/zap_home.png)
+
 ## Usage
 
-You will need a running application to test, which you will want to be as production-like as possible. Before testing, make sure to let your team know, as they may see a spike in requests and errors.
+You will need a running application to test, which you will want to be as production-like as possible. The 'Attack' scripts shouldn't do anything destructive, but will cause a spike in requests and errors. **Inform your team and #devops before running it on live (not locally-hosted) sites.**
 
-1. [Install the ZAP desktop app](https://github.com/zaproxy/zaproxy/wiki/Downloads).
+1. [Install the ZAP desktop app](https://github.com/zaproxy/zaproxy/wiki/Downloads). A recent version of [Java](http://www.oracle.com/technetwork/java/javase/downloads/index.html) (7 or 8 work) is needed to run ZAP.
```

Comment:
Can we remove this new sentence? We should defer to the canonical documentation for any dependencies, and "the Mac OS version includes Java 7" anyway.